docs/entity-cheat-sheet.rst

```diff
@@ -0,0 +1,333 @@
+.. spelling:word-list::
+    hh
+    num
+
+.. comment
+   add an og image like this :og:image: https://docs.getodk.org/_static/img/tutorial-first-form.png
+
+Entity Cheat Sheet
+=================================
+
+This page provides a quick reference for how to design forms in :doc:`XLSForm </tutorial-first-form/>` for ODK that use, create, and update Entities.
+
+Creating and Updating Entity Data
+---------------------------------
+
+When you are designing your form, you can specify how transform submissions into entities. (Learn more about entities :doc:`here </entities-intro/>`.)
```

Comment:
> how transform submissions into entities.

how Entities should be created from submissions

> (Learn more about entities :doc:`here </entities-intro/>`.)

Maybe not necessary if we link above the first time the concept is introduced.